src/psyclone/psyGen.py

```diff
@@ -1604,7 +1602,7 @@ def _get_reductions_list(self, reduction_type):
         result = []
         for call in self.kernels():
             for arg in call.arguments.args:
-                if arg.argument_type in MAPPING_SCALARS.values():
+                if arg.argument_type in MAPPING_SCALARS:
```

Comment:
Please could you update the docstring of this method (formatting and add a :returns: and :rtype:).

Reply:
Done.